x-pack/plugins/apm/public/components/app/correlations/latency_correlations.tsx

```diff
@@ -71,11 +75,45 @@ export function LatencyCorrelations({ onClose }: Props) {
     75
   );
 
-  const { data, status } = useFetcher(
+  const { data: overallData, status: overallStatus } = useFetcher(
     (callApmApi) => {
       if (start && end && hasFieldNames) {
         return callApmApi({
-          endpoint: 'GET /api/apm/correlations/slow_transactions',
+          endpoint: 'GET /api/apm/correlations/latency/overall_distribution',
+          params: {
+            query: {
+              environment,
+              kuery,
+              serviceName,
+              transactionName,
+              transactionType,
+              start,
+              end,
+            },
+          },
+        });
+      }
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Maybe a comment explaining why it is necessary?

Reply:
actually i don't think it's necessary after all since `hasFieldNames` isn't even used anymore in this case